source/guides/network/wifi.rst

```diff
@@ -8,7 +8,9 @@ choose to connect to Wi-Fi while using the
 :ref:`live installer <bare-metal-install-desktop>` image, your Wi-Fi settings
 will be added to your system during the installation process.
 
-NetworkManager provides three simple methods for configuring Wi-Fi.
+NetworkManager provides three simple methods for configuring Wi-Fi: Desktop,
+CLI, and TUI. :command:`wpa_supplicant` is also available if your prefer a
```

Comment:
We should clarify here because NM also uses wpa_supplicant under the hood for wireless connections. 
One idea might be to use headingphrases similar to the static IP doc: https://docs.01.org/clearlinux/latest/guides/network/assign-static-ip.html

- Using NetworkManager
- Using wpa_supplicant + systemd-networkd or Using wpa_supplicant standlone